.github/workflows/checkstyle.yml

```
@@ -0,0 +1,32 @@
+on: 
+  pull_request:
+    branches: ["master", "develop"]
+    paths:
+      - '**/*.java'
+
+jobs:
+  checkstyle:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)